pytest_examples/test_class.py

```diff
@@ -29,3 +29,72 @@ def test_add_floats(self):
     def test_add_floats_and_int(self):
         instance = ClassToTest()
         assert instance.add(1, 2.1) == pytest.approx(3.1)
+
+
+class TestMultiple:
+    def test_multiple_string(self):
+        instance = ClassToTest()
+        with pytest.raises(TypeError):  
+            instance.multiply("2", "3")  
+
+    def test_multiple_arrays(self):
+        instance = ClassToTest()
+        with pytest.raises(TypeError): 
+            instance.multiply([3, 4, 5], 1)
+
+    def test_multiple_ints(self):
+        instance = ClassToTest()
+        assert instance.multiply(2, 3) == 6
+        assert instance.multiply(3, 3) == 9
+        assert instance.multiply(6, 2) == 12
```

Comment:
I think you are missing a really important case in this test. Can you guess what it would be?